image_view/scripts/extract_images_sync

```diff
@@ -33,47 +33,51 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 """Save images of multiple topics with timestamp synchronization.
 
-Usage: rosrun image_view extract_images_sync _inputs:='[<topic_0>, <topic_1>]'
+Usage: ros2 run image_view extract_images_sync _inputs:='[<topic_0>, <topic_1>]'
 """
 
 import sys
-
 import cv2
-
 import cv_bridge
-import message_filters
-import rospy
-from sensor_msgs.msg import Image
+import rclpy
 
+from rclpy.node import Node
+from sensor_msgs.msg import Image
+from message_filters import ApproximateTimeSynchronizer, TimeSynchronizer, Subscriber
 
-class ExtractImagesSync(object):
+class ExtractImagesSync(Node):
 
     def __init__(self):
+        super().__init__('extract_images_sync')
+        self.get_logger().info('Extract_Images_Sync Node has been started')
         self.seq = 0
-        self.fname_fmt = rospy.get_param(
-            '~filename_format', 'frame%04i_%i.jpg')
-        self.do_dynamic_scaling = rospy.get_param(
-            '~do_dynamic_scaling', False)
-        img_topics = rospy.get_param('~inputs', None)
+        self.fname_fmt = self.declare_parameter(
+            'filename_format', 'frame%04i_%i.jpg').value   
+        self.do_dynamic_scaling = self.declare_parameter(
+            'do_dynamic_scaling', False).value
+        img_topics = self.declare_parameter('_inputs', None).value
+
         if img_topics is None:
-            rospy.logwarn("""\
-extract_images_sync: rosparam '~inputs' has not been specified! \
+            self.get_logger().warn("""\
+extract_images_sync: Parameter '_inputs' has not been specified! \
 Typical command-line usage:
-\t$ rosrun image_view extract_images_sync _inputs:=<image_topic>
-\t$ rosrun image_view extract_images_sync \
+\t$ ros2 run image_view extract_images_sync _inputs:=<image_topic>
```

Comment:
These aren't quite right - check out how to use --ros-args (this would actually be interpreted as a remap as written)